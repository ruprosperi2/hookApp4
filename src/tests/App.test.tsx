import { describe, expect, test } from "vitest"
import { render } from "@testing-library/react"
import App from "../App"

// @ts-ignore
globalThis.IS_REACT_ACT_ENVIRONMENT = true;


describe("Testing <App/>", () => {
  test("it should show correctly <App/> component", () => {
    const { container } = render(<App />)
    expect(container).toMatchSnapshot()
  })

})
