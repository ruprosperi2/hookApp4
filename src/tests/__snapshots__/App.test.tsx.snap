// Vitest Snapshot v1

exports[`Testing <App/> > it should show correctly <App/> component 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
